src/styles/performance-dashboard.css

```
@@ -616,6 +616,24 @@ body:has(.fuma-performance-page) .hsas-admin-shell__content {
+}
+
+.fuma-echarts-period-plot > [data-series],
+.fuma-echarts-period-plot > [data-period-date],
+.fuma-echarts-period-plot [data-metric-value] {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)